deployments/kubectl/configMap.yaml

```diff
@@ -112,3 +112,286 @@ data:
+                    }
+                  }
+                ]
+              }
+            },
+            "can_delete": {
+              "union": {
+                "child": [
+                  {
+                    "this": {}
+                  },
+                  {
+                    "computedUserset": {
+                      "relation":"admin"
+                    }
+                  }
+                ]
+              }
+            },
+            "can_edit": {
+              "union": {
+                "child": [
+                  {
+                    "computedUserset": {
+                      "relation":"editor"
+                    }
+                  },
+                  {
+                    "computedUserset": {
+                      "relation":"admin"
+                    }
+                  }
+                ]
+              }
+            },
+            "can_list": {
+              "union": {
+                "child": [
+                  {
+                    "computedUserset": {
+                      "relation":"viewer"
+                    }
+                  },
+                  {
+                    "computedUserset": {
+                      "relation":"admin"
+                    }
+                  }
+                ]
+              }
+            },
+            "can_view": {
+              "union": {
+                "child": [
+                  {
+                    "computedUserset": {
+                      "relation":"viewer"
+                    }
+                  },
+                  {
+                    "computedUserset": {
+                      "relation":"admin"
+                    }
+                  }
+                ]
+              }
+            },
+            "editor": {
+              "this": {}
+            },
+            "viewer": {
+              "union": {
+                "child": [
+                  {
+                    "this": {}
+                  },
+                  {
+                    "computedUserset": {
+                      "relation":"editor"
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          "type":"api"
+        }
+      ]
+    }
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: openfga-init
+data:
+  init.sh: |
+    #!/bin/sh
+
+    set -e
+
+    OPENFGA_URL='http://openfga.default.svc.cluster.local:8080'
+
+    until [ "$(curl -s -o /dev/null -w "%{http_code}" $OPENFGA_URL/stores)" -eq 200 ]; do
+      sleep 5
+    done
+
+    curl -sS -X POST ${OPENFGA_URL}/stores \
+      -H 'Content-Type: application/json' \
+      -d '{"name": "dev"}' | jq -r '.id' > /data/OPENFGA_STORE_ID
+
+    curl -sS -X POST ${OPENFGA_URL}/stores/"$(cat /data/OPENFGA_STORE_ID)"/authorization-models \
+      -H 'Content-Type: application/json' \
+      -d  @/schemas/schema.json | jq -r '.authorization_model_id' > /data/OPENFGA_AUTHORIZATION_MODEL_ID
```

Comment:
Wouldn't it be simpler if we set this as the command in the `openfga-init` job? Something like
```
sh -c 'set -e
OPENFGA_URL='http://localhost:8080'
until [ "$(curl -s -o /dev/null -w "%{http_code}" $OPENFGA_URL/stores)" -eq 200 ]; do
  sleep 5
done
curl -sS -X POST ${OPENFGA_URL}/stores \
  -H 'Content-Type: application/json' \
  -d '{"name": "dev"}' | jq -r '.id' > /data/OPENFGA_STORE_ID
curl -sS -X POST ${OPENFGA_URL}/stores/"$(cat /data/OPENFGA_STORE_ID)"/authorization-models \
  -H 'Content-Type: application/json' \
  -d  @/schemas/schema.json | jq -r '.authorization_model_id' > /data/OPENFGA_AUTHORIZATION_MODEL_ID'
```

Or would it pollute the file too much?

Reply:
I think it's too much if we move the script to the `commands`. Generally, it is a common way to use ConfigMap to share the local script with remote Pod.